subprojects/build-init/src/main/groovy/org/gradle/buildinit/plugins/internal/JavaLibraryProjectInitDescriptor.java

```diff
@@ -44,4 +51,23 @@ protected TemplateOperation testTemplateOperation(BuildInitTestFramework testFra
                 return fromClazzTemplate("javalibrary/LibraryTest.java.template", "test");
         }
     }
+
+    @Override
+    protected Description getDescription() {
+        return DESCRIPTION;
+    }
+
+    @Override
+    protected String getImplementationConfigurationName() {
+        return "implementation";
+    }
+
+    @Override
+    protected void configureBuildScript(BuildScriptBuilder buildScriptBuilder) {
+        buildScriptBuilder.dependency(
+            "api",
+            "This dependency is exposed to consumers",
```

Comment:
and the comment for the `implementation` dependency should also explain a little more about how it is used. The 2 comments should be consistent in what they say.